### PR TITLE
Add Community page that can be accessed from Forum

### PR DESCRIPTION
**Pull request by karenfrilya97**
image

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -112,9 +112,9 @@ public void storeUser(User user) {
   }
 
   /**
-  * Returns the User owned by the email address, or
-  * null if no matching User was found.
-  */
+   * Gets the User owned by the email address, or
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/codeu/data/Datastore.java]: Why this change? In Javadoc, we usually describe methods as "Returns ..." and not "Gets ...".

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -219,4 +225,75 @@ public void storeCommunity(Community community) {
...
+    threadEntity.setProperty("communityId", thread.getCommunityId());
+
+    datastore.put(threadEntity);
+  }
+
+  /**
+   * Gets a list of threads posted in a community, or empty list if the
+   * community has no thread. List is sorted by name ascending.
+   */
+  //TODO: find a better way to sort threads
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/codeu/data/Datastore.java]: Is there some sort of timestamp you could rely on?

**Comment by SuyashLakhotia** [comment]
LGTM overall, just a couple minor changes requested.

If I understand correctly, each community has a list of threads and each thread has a tree of posts?

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 2 above)
Comment (karenfrilya97) [reply src/main/java/com/google/codeu/data/Datastore.java]: I was thinking it should be sorted by popularity. But I'm not sure how to evaluate popularity of a Community. Some of the sorting methods I can think of are:
1. Sort by timestamp which reflects the last time a thread or comment was added to this Community.
2. Sort by the total number of comments across all the threads in a community.
3. Sort by the number of members in a community.

Not sure which is better. Or maybe can use a combination of a few or let the user choose the sorting method?

Anyway I was thinking of adding the sorting feature in a separate PR but do you think I should fix it before merging this PR?

**Comment by karenfrilya97** [comment]
@SuyashLakhotia Yup. But I changed the name to comments instead of posts. 
So at first a thread has one root post which is the main post. But because datastore does not support user-defined classes, I just stored the main post as a text attribute of the thread. Then the contents of the thread more like replies or comments to this main post so I thought Comment is a better name than Post.

And I deleted Post.java from the Forum branch but I forgot to delete it here. I'm replacing it with the Comment.java in PR #32 .

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (karenfrilya97) [reply src/main/java/com/google/codeu/data/Datastore.java]: I followed the verb in the method name but sure I'll change to "Returns ..."

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 2 above)
Comment (SuyashLakhotia) [reply src/main/java/com/google/codeu/data/Datastore.java]: I like Option 1. Its the simplest. Maybe later we can have a sorting feature that allows the user to sort by a specific attribute. Feel free to do it in another PR.

Comment (karenfrilya97) [reply src/main/java/com/google/codeu/data/Datastore.java]: Ok! I'll do it in another PR. I'll add it to the task tracker for now. Thank you!
